scripts/rebase-ledger.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+exit_on_error() {
+    exit_code=$1
+    if [ $exit_code -ne 0 ]; then
+        mv ./data/ledger-temp.json ./data/ledger.json
+        exit $exit_code
+    fi
+}
+
+mv ./data/ledger.json ./data/ledger-temp.json
```

Comment:
Both moving at all, and moving within the git repo, seems error prone as it would unnecessarily dirty the git repo when we're planning to git operations. Consider a _copy_ to a tmp path?

Reply:
accepted